lib/src/flutter_deck_slide.dart

```diff
@@ -250,6 +250,10 @@ class FlutterDeckSlide extends StatelessWidget {
 
   final FlutterDeckThemeData? _theme;
 
+  /// Returns the current theme used by any slide.
+  FlutterDeckThemeData? get theme => _theme;
```

Comment:
Maybe mark this as @visibleForTesting since it probably should not be accessed from the code.